src/components/matching/FeedbackTypeCheckbox.tsx

```diff
@@ -0,0 +1,85 @@
+import styled from '@emotion/styled';
+import { useRef } from 'react';
+
+import type { UseFormRegister } from 'react-hook-form';
+import type { MatchingFeedbackForm } from 'types/matching';
+
+import { BadIcon, EngIcon, FunIcon, NiceIcon } from 'assets/icons';
+
+interface FeedbackTypeCheckboxProps {
+  register: UseFormRegister<MatchingFeedbackForm>;
+}
+
+const FeedbackTypeCheckbox = ({ register }: FeedbackTypeCheckboxProps) => {
+  const { current: FEEDBACK } = useRef([
+    {
+      id: 'isNice',
+      icon: <NiceIcon />,
+      description: '친절해요',
+      hookFormProps: register('feedbackType.isNice'),
+    },
+    {
+      id: 'isFluent',
+      icon: <EngIcon />,
+      description: '영어를 잘해요',
+      hookFormProps: register('feedbackType.isFluent'),
+    },
+    {
+      id: 'isFun',
+      icon: <FunIcon />,
+      description: '재밌어요',
+      hookFormProps: register('feedbackType.isFun'),
+    },
+    {
+      id: 'isBad',
+      icon: <BadIcon />,
+      description: '불쾌해요',
+      hookFormProps: register('feedbackType.isBad'),
+    },
+  ]);
+
+  return (
+    <Container>
+      {FEEDBACK.map((feedback) => {
+        const { id, icon, description, hookFormProps } = feedback;
+        return (
+          <Label key={id}>
+            <input type="checkbox" {...hookFormProps} />
+            {icon}
+            <p>{description}</p>
+          </Label>
```

Comment:
p1;
label에 htmlFor 추가 부탁드립니다.
그리고 label이 inline 레벨이라 내부에 block 레벨인 p태그가 포함되는 것이 적절하지 않습니다.
p태그를 제거해도 좋을 것 같습니다.

```suggestion
          <Fragment key={id}>
            <input type="checkbox" {...hookFormProps} />
            <Label htmlFor={id}>
              {icon}
              {description}
            </Label>
          </Fragment>
```

요렇게 하면 `input:checked` 일 때 스타일 적용이 조금 더 간단해질 것 같습니다!

Reply:
빈태그인 경우 key props를 넣을 수 없다고 생각했는데 Fragment 태그를 사용하면 되었네요!
감사합니다 오늘도 배워갑니다!!

아래의 p3, p1 커맨트도 해당 커밋에서 작업하였습니다!
작업 커밋 : eda74e036119a3668ba3399dd1abdccee02b2eb8